test/dialect/mysql/test_dialect.py

```diff
@@ -320,8 +320,10 @@ def test_mysqlconnector_raise_on_warnings_arg(self):
         [
             "mysql+mysqldb",
             "mysql+pymysql",
+            "mysql+mariadbconnector",
             "mariadb+mysqldb",
             "mariadb+pymysql",
+            "mariadb+mariadbconnector",
```

Comment:
**Michael Bayer** (zzzeek) wrote:

oh great test, OK great

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5505